src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[],
+  creating: Todo | null,
+  onDelete: (todoId: number) => void,
+  inProcessing: number[],
+  updateTodo: (todoId: number [], data: boolean) => void,
```

Comment:
Use `todoId: number` instead of `todoId: number[]` as the parameter type for the `updateTodo` prop.